### DISCUSSION TITLE
Domain is pending verification

### DISCUSSION THREAD

**Discussion by DETAIT** [vercel/community#1460] [Help] [answered]
### Summary

Good day
The approval of my domain name has been pending for more than two weeks, I opened a ticket until now nothing is happening. Can I get in touch with one of the technicians to resolve the matter.

![adminFaspro_domain_issue](https://user-images.githubusercontent.com/122367630/217214520-9d82dca3-59f2-46e6-b503-6257f41e73cd.PNG)

**Comment by swarnava** [comment, marked as answer]
Did you setup the suggested TXT record? Looks like it hasn't propagated yet: https://dnschecker.org/#TXT/_vercel.faspro24.com

**Reply by domainsbasics** [reply]
maybe this can help verify it. Can you try to add the TXT record for faspro24.com and point it at the value they provided  If that still doesn't verify it then try adding the TXT to the subdomain. So you will add a TXT with ADMIN in the host column and point it at that value they provided

**Reply by Green-Dream-Earth** [reply]
i am facing the same error. I tried to add the TXT record along with the value they provided in my domain providers dashboard. Yet it doesn't work as expected. How can i work around this?
